frontend/src/components/ui/blockEditor/ComputationsFileEditor.jsx

```diff
@@ -138,16 +128,22 @@ export default function ComputationsFileEditor({ fetchFileSystem }) {
     setIsLoading(false);
   };
 
-  const handleSave = (e) => {
+  useEffect(() => {
+    if (editorRef.current) {
+      setTimeout(() => {
+        editorRef.current.scrollIntoView({
+          behavior: "smooth",
+          block: "start",
+        });
+      }, 200);
+    }
+  }, [showEditor]);
```

Comment:
Does putting this logic in a `useEffect` change the behaviour?

Reply:
Fixes the scroll issue I sent yesterday, since the timeout was attempting to scroll into view before the component existed